lib/refocus/client.rb

```diff
@@ -19,19 +19,23 @@ def initialize(url:, token:, debug_request: false, debug_response: false)
     end
 
     def subjects
-      Refocus::Subjects.new(url: "#{url}/v1/subjects", token: token)
+      @@subjects ||= Refocus::Subjects.new(url: "#{url}/v1/subjects", token: token)
+      @@subjects
```

Comment:
We don't want this as a class variable - this value will be shared across all instances of the client. It could be an instance variable, just using `@`, if you like, like this:

```ruby
@subjects ||= Refocus::Subjects.new(url: "#{url}/v1/subjects", token: token)
```

I don't think you need a second `@subjects` line - `@subjects` will already be returned by just the first line.